addons/hr_timesheet/migrations/9.0.1.0/tests/test_hr_timesheet.py

```diff
@@ -0,0 +1,11 @@
+# coding: utf-8
+from openerp.tests.common import TransactionCase
+
+
+class TestHrTimesheet(TransactionCase):
+    def test_hr_timesheet(self):
+        self.env.cr.execute(
+            "SELECT line_id FROM hr_analytic_timesheet LIMIT 1")
+        line_id, = self.env.cr.fetchone()
+        line = self.env['account.analytic.line'].browse(line_id)
+        self.assertTrue(line.is_timesheet)
```

Comment:
Thanks for your expansion of the idea. I'm proposing to have the extra test data in the 9.0 branch here: https://github.com/OCA/OpenUpgrade/pull/624